org.eclipse.lsp4e/src/org/eclipse/lsp4e/operations/linkedediting/LSPLinkedEditingReconcilingStrategy.java

```diff
@@ -248,6 +252,24 @@ String getValueInRange(LinkedEditingRanges ranges, VerifyEvent event, int offset
 		return null;
 	}
 
+	private boolean rangesContainOffset(LinkedEditingRanges ranges, int offset) {
+		try {
+			for (Range range : ranges.getRanges()) {
+				if (rangeContainsOffset(range, offset)) {
+					return true;
+				}
+			}
+		} catch (BadLocationException e) {
+			LanguageServerPlugin.logError(e);
+		}
+		return false;
+	}
+
+	private boolean rangeContainsOffset(Range range, int offset) throws BadLocationException {
```

Comment:
I have realized that this method already exists, isn't it the same as `LSPEclipseUtils.isOffsetInRange(int, Range, IDocument)`, could we just use that one?

Reply:
Yes, you're right. Thanks for the catch!